File: src/main/java/com/lawencon/elearn/dto/classhdr/ClassHdrInsertResDto.java
package com.lawencon.elearn.dto.classhdr;

public class ClassHdrInsertResDto {
	private ClassHdrInsertDataResDto data;
	private String message;
	public ClassHdrInsertDataResDto getData() {
		return data;
	}
	public void setData(ClassHdrInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/class-hdr/class-hdr.insert.res.dto.ts
import { ClassHdrInsertDataResDto } from "./class-hdr.insert.data.res.dto";

export class ClassHdrInsertResDto{
    private _data! : ClassHdrInsertDataResDto
	private _message! : string

    public get data(): ClassHdrInsertDataResDto {
        return this._data;
    }

    public set data(data: ClassHdrInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}